packages/yoga/src/Dropdown/native/Dropdown.jsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import styled, { withTheme } from 'styled-components';
+import { arrayOf, func, shape, string, bool } from 'prop-types';
+import { TouchableWithoutFeedback } from 'react-native';
+
+import { Arrow } from '@gympass/yoga-icons';
+
+import Options from './Options';
+import Backdrop from './Backdrop';
+
+const Selector = styled.View`
+  ${({
+    disabled,
+    selected,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    position: relative;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+
+    width: ${dropdown.width}px;
+    padding: ${dropdown.selector.padding.top}px 
+      ${dropdown.selector.padding.right}px 
+      ${dropdown.selector.padding.bottom}px
+      ${dropdown.selector.padding.left}px;
+
+    background-color: ${dropdown.selector.background};
+    border-radius: ${dropdown.selector.border.radius}px;
+    border: 1px solid ${dropdown.selector.border.color};
+    ${
+      disabled
+        ? `border-color: ${dropdown.disabled.selector.border.color};`
+        : ''
+    };
+    ${
+      selected
+        ? `border-color: ${dropdown.selected.selector.border.color};`
+        : ''
+    };
+  `}
+`;
+
+const Label = styled.Text`
+  ${({
+    disabled,
+    selected,
+    theme: {
+      yoga: {
+        components: { dropdown },
+      },
+    },
+  }) => `
+    color: ${dropdown.input.font.color};
+    ${disabled ? `color: ${dropdown.disabled.input.font.color};` : ''}
+    ${selected ? `color: ${dropdown.selected.input.font.color};` : ''}
+  `}
+`;
+
+const getSelectedOption = options => {
+  const filteredOptions = options.filter(item => item.selected === true);
+  return filteredOptions.length > 0 ? filteredOptions[0] : null;
+};
+
+const Dropdown = ({
+  label,
+  disabled,
+  options,
+  cancelActionLabel,
+  confirmActionLabel,
+  onChange,
+  theme: {
+    yoga: {
+      colors: { primary },
+      components: { dropdown },
+    },
+  },
+  ...rest
+}) => {
+  const [selected, setSelected] = useState(getSelectedOption(options));
+  const [isOpen, setIsOpen] = useState(false);
+
+  const iconColor = () => {
+    if (disabled) return dropdown.disabled.arrow.fill;
+    if (selected) return primary[3];
```

Comment:
I can't apply this suggestion... The `dropdown.arrow.fill` will be used just when the arrow isn't  **disabled** or **selected**

Reply:
The primary should be set at the component theme, under our BaseTheme. It would be nice if you always prefer to add style properties into component theme level instead of getting directly from the theme itself.